src/subscription_manager/managercli.py

```diff
@@ -2597,7 +2597,14 @@ def _do_command(self):
 
         if self.options.additions:
             repo_ids = [repo.id for repo in overrides.repo_lib.get_repos(apply_overrides=False)]
-            to_add = [Override(repo, name, value) for repo in self.options.repos for name, value in self.options.additions.items()]
+            to_add = []
+            for name, value in self.options.additions.items():
```

Comment:
iteritems is more efficient as it doesn't need to convert the dict to a list in memory